src/components/Todo/Todo.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import * as cx from 'classnames';
+
+export class Todo extends Component {
+  state = {
+    title: this.props.title,
+    tempTitle: this.props.title,
```

Comment:
no need to write these variables into the state if you need just to read them

Reply:
if you want to change title, just use title from this.props and write changed value to state directly